src/bynder-js-sdk.js

```diff
@@ -1070,9 +1089,9 @@ export default class Bynder {
     uploadFile(file) {
         const { body, filename, data } = file;
         const { brandId } = data;
-        const isBuffer = Buffer.isBuffer(body);
+        const isBufferLike = bufferLike(body);
         const isStream = typeof body.read === 'function';
-        const length = isBuffer ? body.length : file.length;
+        const length = file.length !== undefined ? file.length : body.length;
```

Comment:
Wouldn't it be more readable and consistent like this?
`const length = isBufferLike ? body.length : file.length;`

Reply:
The problem is if you have a browser Blob/File object it doesn't have a `length` property (annoyingly it is called `size` instead). I just extracted `size` and passed it in as the `file.length` but it might be nicer with some additional logic that takes care of that for you. I'll make the change and update the PR.